include/engine/api/route_api.hpp

```diff
@@ -228,42 +228,50 @@ class RouteAPI : public BaseAPI
 
         std::vector<util::json::Object> annotations;
 
-        if (parameters.annotations_type != RouteParameters::AnnotationsType::None)
+        if (parameters.annotations_type != RouteParameters::AnnotationsType::None ||
+            parameters.annotations == true)
         {
+            auto ReqAnnotations = parameters.annotations_type;
+            if ((parameters.annotations == true) & (parameters.annotations_type &
+                RouteParameters::AnnotationsType::None))
```

Comment:
i think it should be `&&` instead of `&` in the first case because it is a boolean comparison?
```
 if ((parameters.annotations == true) && (parameters.annotations_type &
     RouteParameters::AnnotationsType::None))
```

Also I'm not sure what `(parameters.annotations_type & RouteParameters::AnnotationsType::None)` is doing. Is 
`(parameters.annotations_type == RouteParameters::AnnotationsType::None)` not more intuitive and does the same? Not sure though cause I don't know much about bit-magic